package.json

```diff
@@ -3,16 +3,18 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@reduxjs/toolkit": "^1.9.2",
+    "@reduxjs/toolkit": "^1.9.0",
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
```

Comment:
- [ ] 🤔  This project involves connecting an external API to a React-Redux application. You need to connect your React-Redux application to an existing API to create and remove books in a remote server by using Redux Toolkit. Kindly address this and make sure All [Bookstore API](https://www.notion.so/Bookstore-API-51ea269061f849118c65c0a53e88a739) calls are done with `fetch` OR `Axios` .All [Bookstore API](https://www.notion.so/Bookstore-API-51ea269061f849118c65c0a53e88a739) calls are done with [fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch) OR [axios](https://axios-http.com/docs/intro) . 💯 👍🏻

Reply:
In books.js I used the Axios library for handling HTTP requests to an existing  API for my three action creators i.e addBook, removeBook, and getBooks. The state of the books is managed by the bookReducer, which updates the state based on the type of the dispatched action.

Reply:
- [ ] Same thing here, 🤔 I have just check once again but I couldn't see it :man-shrugging: